ejwessel/PrisonersDilemma
# Implement playerChoose()

## Description
Implement the playerChoose() method and test for the Prisoners Dilemma Contract

## Changes 
- Test contract's initial state
- Implement the playerChoose() method
- Add tests for playerChoose(): check requires()
- Add Tests for playerChoose():  Ensuring that data is properly saved to chain

## Tests
```
truffle test
Using network 'test'.

Compiling ./contracts/PrisonersDilemma.sol...

Compilation warnings encountered:

/Users/ejwessel/Dev/PrisonersDilemma/contracts/PrisonersDilemma.sol:44:9: Warning: Unused local variable.
        Player memory player1 = Player(_player1, ActionChoices.NoChoice, 0);
        ^-------------------^
,/Users/ejwessel/Dev/PrisonersDilemma/contracts/PrisonersDilemma.sol:45:9: Warning: Unused local variable.
        Player memory player2 = Player(_player2, ActionChoices.NoChoice, 0);
        ^-------------------^

Deploying contract with:
0x627306090abab3a6e1400e9345bc60c78a8bef57
0xf17f52151ebef6c7334fad080c5704d77216b732


  Contract: PrisonersDilemma
    ✓ Test Initial State of contract (70ms)
    ✓ Test invalid player not in contract (40ms)
    ✓ Test playerChoose() (187ms)


  3 passing (335ms)
```
